charts/rancher-istio/1.1.5/charts/certmanager/templates/deployment.yaml

```diff
@@ -36,7 +36,7 @@ spec:
 {{- end }}
       containers:
       - name: certmanager
-        image: "{{ .Values.hub }}/cert-manager-controller:{{ .Values.tag }}"
+        image: "{{ .Values.global.systemDefaultRegistry }}/{{ .Values.image.repository }}:{{ .Values.image.tag }}"
```

Comment:
We shouldn't do this. When `systemDefaultRegistry ` is empty, it is rendered as "/rancher/image"